### PR TITLE
Merging changes from master branch with local repo

### PR DESCRIPTION
**Pull request by DeepakSumo**
@deborshisaha @rakheegupta Please review

### CODE REVIEW

File: Android/.idea/gradle.xml [hunk 1]
```diff
@@ -3,9 +3,10 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
```
Comment (DeepakSumo) [inline Android/.idea/gradle.xml]: I believe we can delete this file

File: Android/.idea/misc.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
```
Comment (DeepakSumo) [inline Android/.idea/misc.xml]: this too

File: Android/.idea/vcs.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
```
Comment (DeepakSumo) [inline Android/.idea/vcs.xml]: same

**Comment by deborshisaha** [comment]
Find it good.  Marging.

**Pull request merged**